modules/nf-core/bracken/build/main.nf

```diff
@@ -0,0 +1,47 @@
+process BRACKEN_BUILD {
+    tag "$meta.id"
+    label 'process_medium'
```

Comment:
Should this be high? I've only run this command once but seem to recall it requiring a lot of memory

Reply:
Could do, I couldn't remember from when I building some databases... Will bump for security